Dockerfile

```diff
@@ -13,9 +13,10 @@ RUN go build -o mesg-core \
       -ldflags="-X 'github.com/mesg-foundation/core/version.Version=$version'" \
       core/main.go
 
+# NOTE: docker uses xz-utils for uncompress tar.xz files
 FROM ubuntu:18.04
 RUN apt-get update && \
-      apt-get install -y --no-install-recommends ca-certificates=20180409 && \
+      apt-get install -y --no-install-recommends ca-certificates=20180409 xz-utils && \
```

Comment:
is this necessary if we remote the creation of the tarball in the `service.New` function ?

Reply:
yes, it is necessary to extract tar.xz (in service.New we don't create tar with xz compression)